### PR TITLE
feat: add docs & discord button

### PR DESCRIPTION
**Pull request by Fakhri1999**
## Related Issues

#22 

## Before after 
| screen        | before           | after  |
| -------------| -------------| -------------|
| mobile (iPhone 12 Pro) | mocha chroma-gaming xyz_(iPhone 12 Pro) | localhost_3000_(iPhone 12 Pro) |
| tablet (iPad Air)| mocha chroma-gaming xyz_(iPad Air) | localhost_3000_(iPad Air) |
| desktop (HD)| mocha chroma-gaming xyz_(HD) | localhost_3000_(HD) (2) |

### CODE REVIEW

File: src/uikit/Home/HomeContainer.tsx [hunk 1]
```diff
@@ -51,47 +52,64 @@ export function HomeContainer() {
...
+              target='_blank'
+              rel='noreferrer'
+              style={{ width: '100%' }}>
+              <Button w='100%' variant='outline' rounded='full' px={6}>
+                {t('common.home.docs_button_label')}
+              </Button>
+            </a>
+            <a
+              href={publicRuntimeConfig.discordServerUrl}
+              target='_blank'
```
Comment (asaadam) [inline src/uikit/Home/HomeContainer.tsx:106]: afaik if you using `_blank`, the best practice is to use `noopener` along side with it

File: src/uikit/Home/HomeContainer.tsx [hunk 2]
```diff
@@ -51,47 +52,64 @@ export function HomeContainer() {
...
+                position='absolute'
+                right='-90px'
+                top='-15px'
+                transform='rotate(10deg)'>
+                {t('common.home.invite_button_arrow_label')}
+              </Text>
+            </Box>
           </Box>
+          <SimpleGrid columns={{ base: 1, md: 2 }} spacing={2}>
+            <a
```
Comment (asaadam) [inline src/uikit/Home/HomeContainer.tsx:95]: why it's not using `Link` component from chakra ?

File: src/uikit/Home/HomeContainer.tsx [hunk 3]
```diff
@@ -51,47 +52,64 @@ export function HomeContainer() {
...
-            <Text
-              fontSize='lg'
-              fontFamily='Caveat'
-              position='absolute'
-              right='-90px'
-              top='-15px'
-              transform='rotate(10deg)'>
-              {t('common.home.invite_button_arrow_label')}
-            </Text>
+        <Stack direction='column' spacing={2} alignSelf='center'>
```
Comment (asaadam) [inline src/uikit/Home/HomeContainer.tsx:55]: why not use `VStack`, instead of using stack with a direction column?

**Review by asaadam — COMMENTED** [review]
request change

**Comment by Fakhri1999** [comment]
nice suggestion, updated yes @asaadam

**Review by asaadam — APPROVED** [review]

**Pull request merged**